src/main/java/mekanism/common/util/CableUtils.java

```diff
@@ -189,57 +189,30 @@ public static void emit(IEnergyWrapper emitter)
 
 				if(outputtingSides.size() > 0)
 				{
-					double sent = 0;
-					boolean tryAgain = false;
-					int i = 0;
-
-					do {
-						double prev = sent;
-						sent += emit_do(emitter, outputtingSides, energyToSend-sent, tryAgain);
-
-						tryAgain = energyToSend-sent > 0 && sent-prev > 0 && i < 100;
-
-						i++;
-					} while(tryAgain);
+					double sent = emit_do(emitter, outputtingSides, energyToSend);
```

Comment:
Not 100% sure why this was working the way it was, and I might have overlooked something (all of this is done at 5am).
In my tests, this made everything work as expected (and not as it used to), but I obviously can't check everything.
If you think this should stay (in an adapted way to still fix the issue), please leave your feedback.